docs/resources/asserts_custom_model_rules.md

```diff
@@ -0,0 +1,46 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "grafana_asserts_custom_model_rules Resource - terraform-provider-grafana"
+subcategory: "Asserts"
+description: |-
+  Manages Asserts Custom Model Rules through the Grafana API.
+---
+
+# grafana_asserts_custom_model_rules (Resource)
+
+Manages Asserts Custom Model Rules through the Grafana API.
+
+## Example Usage
+
+```terraform
+resource "grafana_asserts_custom_model_rules" "test" {
+  name  = "test-anything"
+  rules = <<-EOT
+    entities:
+    - type: Whatever
+      name: Nothing
+      definedBy:
+      - query: "up{job=\\"nothing\\"}"
```

Comment:
Would it make sense to make this more Terraform native by creating it like 
```
resource "grafana_asserts_custom_model_rules" "test" {
  name  = "test-anything"
  rules {
    entity {
      type = "whatever"
      name = "foo"
      definedBy {
        query = "up{job=\\"nothing\\"}"
      }

      definedBy {
        query = "up{job=\\"nothing\\"}"
      }
    }
  }
}
```

I feel this follows other conventions like in the k8s provider when defining aspects like environment variables as an example